examples/AdvancedOperations/AddSmartDisplayAd.cs

```diff
@@ -310,6 +318,7 @@ private static string CreateBudget(GoogleAdsClient client, long customerId)
         /// <param name="squareMarketingImageAssetResourceName">The square marketing image asset
         /// resource name.</param>
         /// <returns>Resource name of the newly created ad.</returns>
+        // [START AddSmartDisplayAd_3]
```

Comment:
Are the order of these numbers coming from the documentation? Again I'd be concerned about maintenance moving forward: which method is "3" or "4" isn't immediately clear, which I'd think would be made more clear if the tag included the method name.

Reply:
This is highly subjective and it was too expensive to write the script to generate more specific names for the scope of this PR. I am happy to discuss this further offline if needed.

Reply:
Regarding the number ordering: the numbers are incremented as code snippets are found while crawling the guides that display those.